recipes/cp2kdata/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "cp2kdata" %}
+{% set version = "0.6.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/Cp2kData-{{ version }}.tar.gz
+  sha256: 358ec00510e5118af0010d5b669677a53cc803a848e9293a6c7b7806f63993fb
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  entry_points:
+    - cp2kdata = cp2kdata.cli.cmd:cli
+
+requirements:
+  host:
+    - python >=3.8
+    - setuptools
+    - wheel
+    - pip
+  run:
+    - python >=3.8
+    - numpy >=1.24.3
+    - scipy >=1.5.4
+    - matplotlib-base >=3.3.2
+    - ase >=3.20.1
+    - cp2k-input-tools
+    - dpdata
+    - click
+    - regex
+    - monty
+    - pyyaml
+    - asciichartpy
+
+test:
+  imports:
+    - cp2kdata
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/robinzyb/cp2kdata
+  summary: A Small Package to Postprocess Cp2k Output
+  license: LGPL-3.0
```

Should be

```suggestion
  license: LGPL-3.0-only
```

or

```suggestion
  license: LGPL-3.0-or-later
```

See https://spdx.org/licenses/